internal/report/report_test.go

```diff
@@ -33,9 +33,12 @@ type testcase struct {
 	want string
 }
 
-func TestSource(t *testing.T) {
+func TestTextReports(t *testing.T) {
```

Comment:
Could we have a separate test for the folded report? I find such combined meta-tests somewhat complicated... If this is to share setup / teardown code then maybe we can share it by other means?